File: app/src/main/java/com/sample_package_name/sample_application_name/MyApplication.java
package com.sample_package_name.sample_application_name;

import android.app.Application;
import android.content.Context;

import com.facebook.stetho.Stetho;

public class MyApplication extends Application {

    private static Context applicationContext;

    public void onCreate() {
        super.onCreate();
        init();
        initialiseDebugComponents();
    }

    private void init() {
        applicationContext = getApplicationContext();
    }

    private void initialiseDebugComponents() {
        if (BuildConfig.DEBUG) {
            Stetho.initializeWithDefaults(this);
        }
    }

    public static Context getContext() {
        return applicationContext;
    }
}

File: app/src/main/java/com/sample_package_name/sample_application_name/MyApplication.kt
package com.sample_package_name.sample_application_name

import android.app.Application
import android.content.Context
import com.facebook.stetho.Stetho

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        init()
        initialiseDebugComponents()
    }

    private fun init() {
        context = applicationContext
    }

    private fun initialiseDebugComponents() {
        if (BuildConfig.DEBUG) {
            Stetho.initializeWithDefaults(this)
        }
    }

    companion object {
        var context: Context? = null
            private set
    }
}